src/main/java/cs/ualberta/cmput402/tictactoe/TicTacToeGame.java

```diff
@@ -46,11 +71,42 @@ public void playGame(){
         }
 
         board.printBoard();
-        System.out.println("Player " + board.getWinner() + " has won the game!");
+
+        Player winner = board.getWinner();
+        if(winner == null) {
+            this.ties += 1;
+        	System.out.println("Tie Game!");
```

Comment:
Here is also an inconsistent use of tab and space. This tab should be replaced by spaces.

```suggestion
            System.out.println("Tie Game!");
```